---
layout: page
title: Schedule (Fall 2021)
permalink: /schedule-fall2021/
---

We are meeting this semester on Mondays at 8:00-9:00am online. Subscribe to [systematics listserv](/systseminar/listserv/) to receive emails containing connection information (or write to someone listed in [Contact Info](/systseminar/contact-info/)).

We are (at least at the beginning) reviewing some of the talks presented at the Evolution meetings this past summer, the videos of which will be available all semester (and it only costs $10 to gain access at this late date).

### Dec. 6, 2021

From Paul's email: Analisa will discuss a new method for estimating marginal likelihoods that we've been working on in collaboration with our colleagues in the Statistics department. We don't yet have a complete manuscript for you to read as background, so we thought everyone who wanted some background could instead watch the following 15 minute segment of my Phyloseminar Primer on Bayesian phylogenetics, which discusses how marginal likelihoods are estimated and why we might want to estimate them:

Go to [Phyloseminar](http://phyloseminar.org/recorded.html)
Scroll down to Primer part 3b, dated June 27, 2018
The segment to watch starts at 38:40 and goes to 53:45.

Please don't fret about the details of the Steppingstone method presented in the Primer - Analisa will be talking about a totally new method that has nothing to do with steppingstone except that the goal of both is to estimate the marginal likelihood.

### Nov. 29, 2021

(Modified from Kris's email...) Kris would like to discuss the talk by Samuel Hibdige entitled "Widespread lateral gene transfer among grasses":
[https://evolution.econference.io/public/fN8kqCq/main/sessions/18525](https://evolution.econference.io/public/fN8kqCq/main/sessions/18525/102523)

This paper provides more detail about the study referenced in the talk: 

SG Hibdige, P Raimondeau, PA Christin, LT Dunning. 2021. [Widespread lateral gene transfer among grasses.](https://doi.org/10.1111/nph.17328) New Phytologist, 230(6):2474-2486. DOI:10.1111/nph.17328

### Nov. 22, 2021

No class, Thanksgiving break week

### Nov. 15, 2021

Nick says: I am suggesting that we read and discuss this paper by Mason et al. (2020):

[NA Mason, NK Fletcher, BA Gill, WC Funk, and KR Zamudio. 2020. Coalescent-based species delimitation is sensitive to geographic sampling and isolation by distance. Systematics and Biodiversity 18:269-280.]( https://doi.org/10.1080/14772000.2020.1730475)

This paper is highly relevant to my 2nd chapter of graduate work, and to that point I'd like to walk the group through my study system to start off the meeting. Then, we can chat about the paper and then hopefully circle back to relate it to my project, as I will be in California sampling for animals in about a monthâ€”so I'll be able to put all of your advice/thoughts into practice. 

### Nov. 8, 2021

Michelle says: Next Monday I'll share some background and data on whole genome duplications (WGDs) in horseshoe crabs and other chelicerates, with a focus on phylogeny of hox gene clusters. 

Teaser: Horseshoe crabs and spiders/scorpions have each experienced at least 1 round of WGD, and the question as to whether these two groups shared a WGD remains open. Phylogenomic analyses have placed ticks and mites between horseshoe crabs and spiders/scorpions, yet the apparent lack of WGD in ticks/mites has largely been ignored in regards to the shared WGD question. I am attempting to place the tick/mite group in this context using phylogenetic analysis of hox gene clusters.

Although I have a few issues with it, this paper provides some decent context, particularly up to the section "Shared or independent duplications with spiders?":  

[W Nong, Z Qu, Y Li, T Barton-Owen, and A Wong. 2021. Horseshoe crab genomes reveal the evolution of genes and microRNAs after three rounds of whole genome duplication. Commun Biology 4, 83. DOI:10.1038/s42003-020-01637-2](https://doi.org/10.1038/s42003-020-01637-2)

### Nov. 1, 2021

(Modified from Analisa's email...) Analisa would like to discuss the talk by Benjamin Toups entitled "The roots of gene tree variation: do complex models of sequence evolution reduce systematic error" about the effect of model choice on gene tree variation: [https://evolution.econference.io/public/fN8kqCq/main/sessions/18615](https://evolution.econference.io/public/fN8kqCq/main/sessions/18615)

This paper provides more detail about the study referenced in the talk: 

EJ Richards, JM Brown, AJ Barley, RA Chong, RC Thomson. 2018. [Variation across mitochondrial gene trees provides evidence for systematic error: how much gene tree variation is biological?](https://doi.org/10.1093/sysbio/syy013) Systematic Biology 67(5):847-860. DOI:10.1093/sysbio/syy013


### Oct. 25, 2021

Tanner says: On Monday I will present some preliminary data for a group of geometrid moths I am revising. A pair of species in this group show obvious and discrete morphological differences, yet lack discernible genetic differences across both nuclear and mitochondrial markers. I'll introduce this handsome group and then present the taxonomic conundrum I appear to be faced with.

Please find attached an older paper that looks to be situationally similar. My intention for choosing this paper is its relevance to our broader discussion, not necessarily the molecular methods ;)

[Nice, CC, and AM Shapiro. 1999. Molecular and morphological divergence in the butterfly genus Lycaeides (Lepidoptera: Lycaenidae) in North America: evidence of recent speciation. J. Evol. Biol. 12:936-950. DOI:10.1046/j.1420-9101.1999.00111.x](https://doi.org/10.1046/j.1420-9101.1999.00111.x)

### Oct. 18, 2021

Eric Schultz has a trait evolution problem that he wants to share with us this week and get our feedback on the best way to address the question. 

The trait is euryhalinity, and Eric suggested this paper as background:
[Hutchinson, GE. 1960. On Evolutionary Euryhalinity. American Journal of Science 258A:98-103.](http://earth.geology.yale.edu/~ajs/1960/ajs_258A_11.pdf/98.pdf)

### Oct. 11, 2021

Paul thought this talk describing an interesting and different approach to evaluating the appropriateness of different models for continuous morphological traits would be an interesting topic for discussion: [Teisha King: Identifying atypical modes of continuous trait evolution](https://evolution.econference.io/public/fN8kqCq/main/sessions/18550/102638)

This applet provides an easy way to learn about the difference between the BM and OU models that feature in this talk: [Ornstein-Uhlenbeck vs. Brownian motion model](https://plewis.github.io/applets/ou/)

You also need to know a little bit about phylogenetically independent contrasts (PIC) to understand this talk. Here is the original classic paper that introduced PIC (and also founded the field of phylogenetic comparative methods): [J Felsenstein. 1985. Phylogenies and the comparative method. The American Naturalist 125:1-15.](https://doi.org/10.1086/284325). You don't need to understand all parts of this paper in order to understand the talk. All you really need to know is what a phylogenetic independent contrast is: a measure of how different a trait is between two sister taxa. The math in this paper is all about handling the fact that there is some extra uncertainty associated with ancestral reconstructions of trait values deeper in the tree because you have not actually observed those ancestral trait values, and the amount of uncertainty grows with how far the ancestral node is from the tips of the tree (where you _can_ observe trait values).

### Oct. 4, 2021

Zach will be talking about cotemporal diversification across independent lineages to infer shared evolutionary events. You can find the link the talk here: [Jamie Oaks: Expanding the space of phylogenetic trees to infer patterns predicted by shared processes of diversification](https://evolution.econference.io/public/fN8kqCq/main/sessions/18480/99926)

Additional reading for this talk can be found in the preprint linked here: [doi.org/10.1101/2021.07.23.453597](https://doi.org/10.1101/2021.07.23.453597)

### Sep. 27, 2021

From Mark Stukel: For Monday's reading/viewing, I have chosen a talk about using the Fossilized Birth-Death process to improve uncertainty in fossil ages. It nicely follows up on our discussion last time about uncertainty in fossil ages as well as fossil placement. Here is the talk: 
[https://evolution.econference.io/public/fN8kqCq/main/sessions/18480/98885](https://evolution.econference.io/public/fN8kqCq/main/sessions/18480/98885)

For background reading, I think it would be useful to read the original paper outlining the Fossilized Birth-Death method, which is linked here: [https://doi.org/10.1073/pnas.1319091111](https://doi.org/10.1073/pnas.1319091111)

### Sep. 20, 2021

Let's take a look this week at the talk "A new method for inferring large-scale time-calibrated species trees" presented by Deren Eaton. To see the talk, you will need to register for the meetings at https://evolutionmeetings.org (scroll down to the section entitled "Evolution 2021 post-conference registration is  still open!" and click the "here" link in that paragraph). 

This talk mentions the following review paper, which would be excellent background reading for this week:

GP Tiley, JW Poelstra, M dos Reis, Z Yang, and A Yoder. 2020. Molecular clocks without rocks: new solutions for old problems. Trends in Genetics 36(11):845-856. [DOI:10.1016/j.tig.2020.06.002](https://doi.org/10.1016/j.tig.2020.06.002)

### Sep. 13, 2021

EEB alum Jack Phillips joined us from Utah (at the brutal hour of 6am!) to answer questions about his talk entitled "Lung evolution in anuran tadpoles". The main question is whether moving from ponds to streams predisposes tadpoles to "lose" their lungs, which really means delay their lung development until metamorphosis, due to the increased risk of being swept away by the current due to the increased buoyancy provided by inflated lungs.

Some background on the methods Jack used can be found in the [BayesTraits manual](http://www.evolution.rdg.ac.uk/BayesTraitsV3.0.5/BayesTraitsV3.0.5.html), specifically pages 26-32 corresponding to sections "Discrete" through "Discrete:Covarion". 
